tests/integration/sqs_work_dispatcher_tests.py

```diff
@@ -1033,36 +1033,42 @@ def test_cleanup_second_try_succeeds_after_killing_worker_with_dlq(self):
         queue = sqs_queue()
         queue.send_message(MessageBody=msg_body)
         worker_sleep_interval = 0.05  # how long to "work"
+        cleanup_timeout = int(worker_sleep_interval + 3)  # how long to allow cleanup to run, max (integer seconds)
```

Comment:
Can we move this comment up? It's not officially in our style guide yet but what we generally do.

Reply:
Can we leave it as-is?
- There's probably 100s of cases in this file that have comments on the same line of the code that would need to be fixed as well.
- Being that it's a unit tests, comment readability probably isn't super critical
- It can be helpful too to keep the flow of the code uninterrupted.

Reply:
I can leave it in for now but we will most likely put it in our style guide soon and resolve them in a tech improvement.